src/pages/behavioral-competencies-page/BehavioralCompetenciesForm.tsx

```diff
@@ -93,7 +95,14 @@ export function BehavioralCompetenciesForm(props: Props) {
                 details={<BehavioralCompetencyHelpText criteria="sahabiness" isSelfReview={isSelfReview} />}
                 prefix="sahabiness"
               />
-              <Guide title={i18n._('Organization Culture Adoption')} guideText={selfReviewSahabinessHelpModalText} />
+              {selfReviewSahabinessHelpModalText ? (
```

Comment:
Checking whether there is a non-empty text before showing the button was a nice touch! 👍